token/js/src/state/account.ts

```diff
@@ -82,23 +82,7 @@ export const AccountLayout = struct<RawAccount>([
 /** Byte length of a token account */
 export const ACCOUNT_SIZE = AccountLayout.span;
 
-/**
- * Retrieve information about a token account
- *
- * @param connection Connection to use
- * @param address    Token account
- * @param commitment Desired level of commitment for querying the state
- * @param programId  SPL Token program account
- *
- * @return Token account information
- */
-export async function getAccount(
-    connection: Connection,
-    address: PublicKey,
-    commitment?: Commitment,
-    programId = TOKEN_PROGRAM_ID
-): Promise<Account> {
-    const info = await connection.getAccountInfo(address, commitment);
+function unpackTokenAccount(info: AccountInfo<Buffer> | null, address: PublicKey, programId: PublicKey) {
```

Comment:
nits: possible to call this `unpackAccount` for consistency, and move it down to the bottom of the file since it's an internal function?